components/module-components/create-blogs/BlogListView.tsx

```diff
@@ -0,0 +1,98 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import Button from '../../reusable-components/button/Button';
+import ReusableTable from '../../reusable-components/reusable-table/ReusableTable';
+import Filter from '../SearchAndFilter/Filter';
+import Search from '../SearchAndFilter/Search';
+import ActionDropdown from '../../reusable-components/reusable-table/ActionDropdown';
+import style from './createBlogs.module.scss';
+
+interface BlogPost {
+  id: string;
+  title: string;
+  tags: { name: string }[];
+  author: string;
+}
+
+const BlogListView = () => {
+  const [data, setData] = useState<BlogPost[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('https://staging-api.learnhub.mk/blog-posts');
```

Comment:
Because of security and maintainability reasons, it is not a good practice to hardcore URLs. 
You should use process.env.(the value here)
We have added the https://staging-api.learnhub.mk as BASE_URL i think.